server/src/helper/product-helper.js

```diff
@@ -0,0 +1,45 @@
+const Validation = {
+  validateName(validatedProduct) {
+    if (!validatedProduct.value.name) {
+      validatedProduct.errors.push('"name" is required');
+      return validatedProduct;
+    }
+    if (validatedProduct.value.name.length < 5) {
+      validatedProduct.errors.push('"name" length must be at least 5 characters long');
+    }
+    if (validatedProduct.value.name.length > 15) {
+      validatedProduct.errors.push('"name" length must be at most 15 characters long');
+    }
+    return validatedProduct;
+  },
+  validatePrice(validatedProduct) {
+    if (!validatedProduct.value.price) {
+      validatedProduct.errors.push('"price" is required');
+      return validatedProduct;
+    }
+    if (validatedProduct.value.price < 1000) {
+      validatedProduct.errors.push('"price" must be larger than or equal to 1000');
```

Comment:
Any reason why the price should be less than 1000?

Reply:
The price should be more than 1000, because on the front-end, no product is less than 1000
![screenshot from 2018-10-24 10-52-25](https://user-images.githubusercontent.com/6943256/47422636-01720900-d77b-11e8-9442-9bedc399426a.png)